dataprofiler/labelers/base_data_labeler.py

```diff
@@ -267,23 +273,29 @@ def set_labels(self, labels):
 
     def predict(
         self,
-        data,
-        batch_size=32,
-        predict_options=None,
-        error_on_mismatch=False,
-        verbose=1,
-    ):
+        data: Union[pd.DataFrame, pd.Series, np.ndarray],
+        batch_size: int = 32,
+        predict_options: Dict[str, bool] = None,
+        error_on_mismatch: bool = False,
+        verbose: bool = True,
```

Comment:
I changed the default of `verbose` from `1` to `True` because it is only used in `self._model.predict()`, which would takes in a bool.

Reply:
nice